### ISSUE TITLE
Recording of mouse motion

### ISSUE THREAD

**Issue by JFD23** [elan-ev/opencast-studio#577]
it would be great to see the mouse moving and pointing during screenrecording. It is a behaviour users are familiar with from other tools.

**Comment by LukasKalbertodt** [comment]
Yep, it would be great indeed. Sadly, the browser support for this feature is currently really bad (only Opera supports it). This means we can't do anything about it, until the browsers implement it. Also see this issue: https://github.com/elan-ev/opencast-studio/issues/185

**Comment by JFD23** [comment]
I just searched for mouse and maus in the issue history before opening this one 🤦

**Comment by LukasKalbertodt** [comment]
FYI: recording the cursor already seems to work fine on Chrome (Ubuntu and Windows). Firefox does indeed not support it yet. But yeah, the `caniuse` data is outdated as it seems...

**Comment by JFD23** [comment]
A colleague reported that it is also working on win10 with Edge V81.
